daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/PartialTransaction.scala

```diff
@@ -225,13 +224,14 @@ private[speedy] case class PartialTransaction(
     context: PartialTransaction.Context,
     aborted: Option[Tx.TransactionError],
     contractState: ContractStateMachine[NodeId]#State,
-    localContracts: Set[Value.ContractId],
     actionNodeLocations: BackStack[Option[Location]],
 ) {
 
   import PartialTransaction._
 
-  def consumedBy: Map[Value.ContractId, NodeId] = contractState.activeState.consumedBy
+  def consumedByOrInactive(cid: Value.ContractId): Option[Either[NodeId, Unit]] = {
```

Comment:
probably worth introducing a proper trait for this but let’s not do it in this pr.